man/adoc/bpftrace.adoc

```diff
@@ -675,7 +675,15 @@ The following operators are available for integer arithmetic:
 
 |===
 
-// TODO: Words about integer conversion when types mismatch
+Operations between a signed and an unsigned integer are allowed providing
+bpftrace can statically prove a safe conversion is possible. If safe conversion
+is not guaranteed, the operation is undefined behavior and a corresponding
+warning will be emitted.
```

Comment:
Do we mean error here instead of warning?

Reply:
No, mixing signed and unsigned currently emits a warning:

```
$ sudo ./build/src/bpftrace -e 'BEGIN { $s = -5; $u = (uint64) 5; $s == $u }'
stdin:1:38-40: WARNING: comparison of integers of different signs: 'int64' and 'uint64' can lead to undefined behavior
BEGIN { $s = -5; $u = (uint64) 5; $s == $u }
                                     ~~
Attaching 1 probe...
^C
```